my-playbook/group_vars/development/all.yml

```diff
@@ -0,0 +1,9 @@
+# my-playbook/group_vars/development/all.yml
```

Comment:
Why the variable file naming and inventory are using development and dev repectively, please keep it in sync, as discussed!

Reply:
I have renamed the variable file name as we discussed